Lib/test/test_asyncio/test_events.py

```diff
@@ -2865,6 +2865,19 @@ class TestCGetEventLoop(GetEventLoopTestsMixin, unittest.TestCase):
         get_running_loop_impl = events._c_get_running_loop
         get_event_loop_impl = events._c_get_event_loop
 
+class TestServer(unittest.TestCase):
+
+    def test_get_loop(self):
+        loop = asyncio.new_event_loop()
+        proto = MyProto(loop)
+        server = loop.run_until_complete(loop.create_server(lambda: proto, '0.0.0.0', 0))
+        self.assertEqual(server.get_loop(), loop)
+        loop.close()
```

Comment:
This test will leak an open server object. Add `server.close(); loop.run_until_complete(server.wait_closed())`